rust/operator-binary/src/backend/mod.rs

```diff
@@ -182,6 +182,14 @@ impl SecretVolumeSelector {
     ) -> Result<Vec<Address>, ScopeAddressesError> {
         use scope_addresses_error::*;
         let cluster_domain = &pod_info.kubernetes_cluster_domain;
+
+        // TODO: Docs!!!
+
+        let mut cluster_domains = vec![cluster_domain.to_string()];
+        if let Some(cluster_domain_without_trailing_dot) = cluster_domain.strip_suffix('.') {
+            cluster_domains.push(cluster_domain_without_trailing_dot.to_owned());
+        }
```

Comment:
In my testing, Kerberos wants consistency and TLS doesn't really care. Either *should* be helped by doing both.

Reply:
Tried to respond in https://github.com/stackabletech/secret-operator/pull/547#issuecomment-2605005175